src/main/scala/org/broadinstitute/dsde/rawls/openam/RawlsOpenAmClient.scala

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.dsde.rawls.openam
+
+import org.broadinstitute.dsde.rawls.openam.UserInfoResponseProtocol._
+import org.broadinstitute.dsde.vault.common.openam.OpenAMClient
+import org.broadinstitute.dsde.vault.common.openam.OpenAMResponse._
+import spray.client.pipelining._
+import spray.http.{MediaTypes, HttpEntity, Uri}
+import spray.httpx.SprayJsonSupport._
+
+import scala.concurrent.Future
+
+/**
+ * Wrapper/replacement for vault-common's OpenAMClient that has the features we want
+ * (Rawls config, email extraction, etc)
+ */
+object RawlsOpenAmClient {
```

Comment:
I don't really like using object in this context. It makes the code less pluggable. For example, there is no way to replace RawlsOpenAmClient with a mock implementation. An alternative would be to make this a trait then mix it in where required.